### DISCUSSION TITLE
How to preview the page if I want to contribute?

### DISCUSSION THREAD

**Discussion by lqhl** [superlinked/VectorHub#130] [Discussions]
I'm wondering if it's possible for me to preview the web page locally before submitting a pull request.

**Comment by svonava** [comment]
Hi @lqhl - this is tricky before we opensource the frontend (work in progress), in the meantime, please make your best guess using the vendor.schema.json and the json files that already exist and we will give you feedback if something looks off.

Does that work?

**Reply by lqhl** [reply]
no problems. thank you.
